static/resources.json

```diff
@@ -54,6 +54,12 @@
         "url": "https://www.sublimetext.com/",
         "payment": "optional",
         "payment_description": "Nagware; will ask you to buy the full version after every X saves"
+      },
+      "Atom (Free)": {
+          "description": "A free Electron based editor, a \"hackable text editor for the 21st century\".",
```

Comment:
Should be `Electron-based`, not `Electron based`

Reply:
Perhaps also mention that Atom is maintained by the GitHub team - at least, I think that's the case, right?

Reply:
It is

(oh now it is alphabetic :stuck_out_tongue:)